<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAnalysis</class>
 <widget class="QWidget" name="FormAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>900</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/Res/layout3.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_TICA3110">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/UICtrls/Res/temperature_controller.png</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_TICA3110">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_TICA3110">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>550</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>背压阀至总收集管路伴热</string>
   </property>
   <property name="text">
    <string>TICA3110</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_TICA3120">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>590</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/UICtrls/Res/temperature_controller.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_TICA3130">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>630</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/UICtrls/Res/temperature_controller.png</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_TICA3120">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>590</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_TICA3130">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>630</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_TICA3120">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>620</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>背压阀板加热</string>
   </property>
   <property name="text">
    <string>TICA3120</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_TICA3130">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>660</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>GC1管路伴热</string>
   </property>
   <property name="text">
    <string>TICA3130</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_GC1">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>670</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/UICtrls/Res/gc.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_TICA3140">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>620</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/UICtrls/Res/temperature_controller.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_GC2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>670</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/ui_ctrls/UICtrls/Res/gc.png</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_TICA3140">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>585</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txt_label_TICA3140">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>650</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>GC2管路伴热</string>
   </property>
   <property name="text">
    <string>TICA3140</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
